account_ux/wizards/account_change_currency.py

```diff
@@ -54,6 +54,8 @@ def onchange_currency(self):
                 1.0, self.currency_to_id, self.move_id.company_id,
                 date=self.move_id.date or
                 fields.Date.context_today(self))
+            if self.move_id.company_currency_id == self.currency_to_id:
+                self.move_id.l10n_ar_currency_rate = 0
```

Comment:
field l10n_ar_currency_rate does not exist in this module.

Reply:
also, review if actually need to change it to 0. should not be 1.0 the correct rate? this on is already computed in the line 53